src/components/Contacts/ContactDetails/ContactDetails.graphql

```diff
@@ -0,0 +1,59 @@
+fragment ContactDetails on Contact {
+  id
+  ...ContactRow
+  avatar
+  churchName
```

Comment:
What are you loading data for here? This looks like a lot of fields that might not be used everywhere. The header? Header and the first tab (Tasks)? I think the contact details tab should be lazy loaded. (To be clear I mean the contact details tab in the contact details page. Maybe we should name those 2 components differently somehow.)

Gyasi wrote a query for the tasks tab already https://github.com/CruGlobal/mpdx-react/pull/47

Reply:
I didn't realize Gyasi made a query for the Tasks Tab, We could probably create separate data for each tab and make the query only when the tab is opened.